mms/model_loader.py

```diff
@@ -43,10 +44,21 @@ def load(model_dir, handler):
         """
         manifest = None
         manifest_file = os.path.join(model_dir, MANIFEST_FILENAME)
+
+        # Legacy manifest handling
+        manifest_legacy = None
+        manifest_legacy_file = os.path.join(model_dir, MANIFEST_LEGACY_FILE)
+
+        if os.path.isfile(manifest_legacy_file):
+            try:
+                manifest_legacy = json.load(open(manifest_legacy_file))
+            except ValueError as e:
+                logger.info(repr(e))
```

Comment:
This shouldn't be info it should be Exception. Can we assume service code will work without the legacy code?

Reply:
Oh right. It might not work without legacy files. I see your point. Will raise an exception here.

Reply:
Done.